cluster-autoscaler/cloudprovider/ovhcloud/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
+- Kafei59
```

Comment:
You cannot be an owner unless you belong to the Kubernetes org. You can either:
- Get yourself accepted to the Kubernetes org.
- Comment yourself from OWNERS
- Find a different owner.

Reply:
@mwielgus, yes I have commented this above.
Get accepted to the Kubernetes org looks pretty complicated, since I do not know any sponsors and have only contributed "a little" for now on.
What shoud I do ?

Reply:
Sorry for typo. I meant comment out. Or just remove the owners file altogether.